riggerlib/client.py

```diff
@@ -115,6 +115,7 @@ def terminate(self):
         try:
             self._request({'event_name': 'shutdown'})
             self.ready = False
+            self.zmq.close()
```

Comment:
the zmq socket is opened as a thread local

so only one of them would be closed in case another thread opens as well